ioeffect/src/main/scala/scalaz/ioeffect/MonadIO.scala

```diff
@@ -12,16 +14,33 @@ import scalaz.Monad
  *
  * @tparam M - the monad in which to lift
  */
-trait MonadIO[M[_]] extends Monad[M] {
+trait MonadIO[M[_], E] {
```

Comment:
missing E in the docs.

You write docs, you create a liability for the next person :stuck_out_tongue: to be wrong.

Reply:
FFFFFFFFFFF yeah I'll amend the docs. Good catch.